npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -242,10 +241,10 @@ func ruleExists(ports []networkingv1.NetworkPolicyPort, peer []networkingv1.Netw
 }
 
 // peerAndPortRule deals with composite rules including ports and peers
-// (e.g., IPBlock, podSelector, namespaceSelector, or both podSelector and namespaceSelector)
-func peerAndPortRule(npmNetPol *policies.NPMNetworkPolicy, ports []networkingv1.NetworkPolicyPort, setInfo []policies.SetInfo) {
+// (e.g., IPBlock, podSelector, namespaceSelector, or both podSelector and namespaceSelector).
+func peerAndPortRule(npmNetPol *policies.NPMNetworkPolicy, direction policies.Direction, ports []networkingv1.NetworkPolicyPort, setInfo []policies.SetInfo) {
```

Comment:
Good catch. 
I will walk through dstlist and srclist.

Yes. Confirmed that ports are always for destination.

Reply:
Revised them now with a method.

However, it seems we can do better job with only one `PeerList` instead of having two `SrcList` and `DstList`. 
I put the comment about it, but want to do it next PR even though it makes sense since it needs to change in multiple places. wdyt?